packages/next/src/client/components/react-dev-overlay/internal/components/copy-button/index.tsx

```diff
@@ -0,0 +1,74 @@
+import { useState } from 'react'
+
+export function CopyButton({
+  label,
+  successLabel,
+  content,
+  ...props
+}: React.HTMLProps<HTMLSpanElement> & {
+  label: string
+  successLabel: string
+  content: string
+}) {
+  // copied: 0 = not copied, 1 = copied, 2 = error
```

Comment:
This is kinda confusing to me 😆 
Any reason to not use an enum? `'initial' | 'success' | 'error'`